src/Task.ts

```diff
@@ -77,6 +80,7 @@ export class Task {
         doneDate,
         recurrence,
         blockLink,
+        reminderTime, //reminder attribute, same as dueDate but with time
```

Comment:
I might rename this to reminderDateTime, to make it clear that it includes a date as well. Not sure about that though.

In general, it's good to make the names clear enough that comments are not needed, especially in view of how many times this comment is repeated further down.